docs/system/cpu-metrics.md

```diff
@@ -33,29 +33,7 @@ This metric is [recommended][MetricRecommended].
 
 | Name     | Instrument Type | Unit (UCUM) | Description    | Stability | Entity Associations |
 | -------- | --------------- | ----------- | -------------- | --------- | ------ |
-| `cpu.time` | Counter | `s` | Seconds each logical CPU spent on each mode | ![Development](https://img.shields.io/badge/-development-blue) |  |
-
```

Comment:
could you please update md to generate correct metric here?

Reply:
Since there are no more `cpu.*` metrics, I've removed this document in [f5597bc](https://github.com/open-telemetry/semantic-conventions/pull/2295/commits/f5597bcf958db79ee40ef55c84e47c950c2bf0b4)

Reply:
do we have `system.cpu.time` metrics somewhere in markdown? It doesn't seem so